docs/architecture.md

```diff
@@ -75,3 +116,114 @@ the decision.
 ### Attestation diagram
 
 ![attestation](images/Fobnail-flows-attestation.png)
+
+---
+
+## Keys and certificates
+
+### Platform Owner
+
+#### Knows in advance
+
+- Platform Owner key (signing and encryption)
+  - Fobnail and remote platform provisioning
+  - used for TLS and for signing certificates
+- Platform Owner certificate
+  - Fobnail and remote platform provisioning
+  - has to be able to provide whole CA chain
+  - both Fobnail Token and Platform must trust root CA in this chain
```

Comment:
@krystian-hebel This does not render properly (affects entire list)
![Screenshot_20220328_103235](https://user-images.githubusercontent.com/44615573/160358587-2b2d8704-83fc-4611-aee1-c906701d87c0.png)

Reply:
Fixed